### DISCUSSION TITLE
Add Weather Radar Element

### DISCUSSION THREAD

**Discussion by vednolacni** [andrejs2/arso-vremenska-integracija#4] [Ideas]
It would be fantastic to have a weather radar element integrated into our dashboard. This feature could enhance real-time weather monitoring and provide a visually intuitive way for users to see current weather patterns, similar to the implementation in [this weather-radar-card project](https://github.com/Makin-Things/weather-radar-card).

**Comment by andrejs2** [comment]
Well, I use generic [camera integration](https://www.home-assistant.io/integrations/generic/) for rain radar and I've put this link: https://meteo.arso.gov.si/uploads/probase/www/observ/radar/si0-rm-anim.gif 
Then I use camera card to show rain radar on a dashboard.

But this functionality is also on my bucket list for integration. Lack of time and I'm currently busy with Voice Assist of HA... The drawback is that the integration has to be rebuild because now it's build as part of [weather integration](https://www.home-assistant.io/integrations/weather/) and for this any maybe any other features like pm2.5, pm10, pollen, UV index, bio weather etc. it should be build differently. 

Thanks for the idea, in the meantime you can use generic camera integration.

**Reply by vednolacni** [reply]
Using the generic camera integration and a camera card to display a rain radar on your dashboard is a really clever solution. 

Thanks!
